src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java

```diff
@@ -22,19 +27,22 @@
     public void actionPerformed(@NotNull AnActionEvent e) {
         project = LibertyProjectUtil.getProject(e.getDataContext());
         if (project == null) {
-            log.debug("Unable to " + actionCmd + ", could not resolve project");
+            String msg = "Unable to " + actionCmd + ", could not resolve project. Ensure you run the Liberty action from the Liberty tool window";
+            notifyError(msg);
+            log.debug(msg);
             return;
         }
 
         buildFile = (VirtualFile) e.getDataContext().getData(Constants.LIBERTY_BUILD_FILE);
         if (buildFile == null) {
-            log.debug("Unable to " + actionCmd + ", could not resolve configuration file for  " + project.getName());
+            String msg = "Unable to " + actionCmd + ", could not resolve configuration file for  " + project.getName() + ". Ensure you run the Liberty action from the Liberty tool window";
```

Comment:
Change to something like this to avoid comma splice:
`"Unable to " + actionCmd + ": Could not resolve configuration file for " + project.getName() + ".`